src/scripts/main.js

```diff
@@ -1,3 +1,245 @@
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  h2.textContent = title;
+  p.textContent = description;
+  notification.append(h2, p);
+
+  body.appendChild(notification);
+  setTimeout(() => (notification.style.display = 'none'), 2000);
+};
+
+button.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const formData = new FormData(form);
+  const data = Object.fromEntries(formData.entries());
+
+  const isValide = () => {
+    const checkInput = Object.values(data).every((val) => val !== '');
+    const checkName = data.name.length >= 4;
+    const checkAge = data.age >= 18 && data.age <= 90;
+
+    if (checkInput && checkName && checkAge) {
+      return true;
+    }
+
+    return false;
+  };
+
+  if (isValide()) {
+    createRow(data);
+    pushNotification(10, 'Success', 'Data added successfully', 'success');
+    form.reset();
+  } else {
+    pushNotification(
+      10,
+      'Error',
+      'Please fill in all fields and check valid values',
+      'error',
+    ); // eslint-disable-line
+  }
+});
+
+// DBLCLICK
+
+// eslint-disable-next-line no-shadow
+// tbody.addEventListener('dblclick', (event) => {
+//   const cell = event.target.closest('td');
+
+//   if (cell) {
+//     const copyTextCell = cell.textContent;
+//     const input = document.createElement('input');
+//     const cellIndex = cell.cellIndex;
+//     const cellTextChange = (value) => {
+//       const checkInputValue = () => {
+//         if (value.length) {
+//           if (cellIndex === 4) {
+//             input.value = `$${toNumber(value).toLocaleString('en-US')}`;
+//           }
+//         } else {
+//           input.value = copyTextCell;
+//         }
+
+//         return input.value;
+//       };
+
+//       cell.innerText = checkInputValue(input.value);
+//       input.remove();
+//     };
+
+//     input.type = 'text';
+//     input.classList.add('cell-input');
+//     cell.innerText = '';
+//     cell.appendChild(input);
+
+//     input.addEventListener('blur', () => {
+//       cellTextChange(input.value);
+//     });
+
+//     // eslint-disable-next-line no-shadow
+//     window.addEventListener('keydown', (event) => {
+//       if (event.key === 'Enter') {
+//         cellTextChange(input.value);
+//       }
+//     });
+//   }
+// });
+// eslint-disable-next-line no-shadow
+tbody.addEventListener('dblclick', (event) => {
+  const cell = event.target.closest('td');
+
+  if (cell && cell.cellIndex !== undefined) {
+    const initialValue = cell.textContent.trim();
+    const input = document.createElement('input');
+
+    input.type = 'text';
+    // input.value = initialValue.replace(/[$,]/g, '');
+    input.classList.add('cell-input');
+    cell.innerHTML = '';
+    cell.appendChild(input);
+    input.focus();
+
+    const saveValue = () => {
+      const newValue = input.value.trim();
+
+      if (cell.cellIndex === 4 && isNaN(+newValue)) {
```

Comment:
The condition `isNaN(+newValue)` might not be sufficient for salary validation. Consider ensuring that `newValue` is a valid number and properly formatted as a currency.